bin/common.sh

```diff
@@ -187,9 +187,42 @@ function randomPassword {
   date +%s | sha256sum | base64 | head -c 32 ; echo
 }
 
+function disable_sa_token_automount {
+  local ns sa_name should_disable
+  ns=$1
+  sa_name=$2
+  should_disable=${SEC_DISABLE_SA_TOKEN_AUTOMOUNT:-true}
+
+  if [ "$should_disable" == "true" ]; then
+     log_debug "Disabling automount of API tokens for serviceAccount [$ns/$sa_name]"
```

Comment:
Add a check that the ServiceAccount actually exists and exit w/WARNING if not?  Or, let it ERROR out?